docs/planning/backend/app/api/imports.py

```
@@ -0,0 +1,144 @@
+            status_code=429,
+            detail=f"Fragella API quota exhausted. {client.requests_remaining} requests remaining this month."
+        )
+
+    try:
+        results = client.search(query, limit)
+        return {
+            "query": query,
+            "results": results,
+            "requests_remaining": client.requests_remaining
+        }
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)